packages/vm/src/modules/mod.rs

```diff
@@ -5,7 +5,8 @@ mod pinned_memory_cache;
 mod versioning;
 
 pub use cached_module::CachedModule;
-pub use file_system_cache::FileSystemCache;
+#[allow(unused_imports)]
+pub use file_system_cache::{FileSystemCache, NewFileSystemCacheError};
```

Comment:
This was fixed on main, no diff needed here (https://github.com/CosmWasm/cosmwasm/pull/1994/files)